zerver/openapi/markdown_extension.py

```diff
@@ -441,6 +441,13 @@ def render_description(self, function: str) -> List[str]:
         path, method = function.rsplit(':', 1)
         description_dict = get_openapi_description(path, method)
         description_dict = description_dict.replace('{{api_url}}', self.api_url)
+        if'{!api-admin-only.md!}' in description_dict:
+            file_content = ''
+            file_name = os.path.join(os.path.dirname(__file__),
+                                     '../../templates/zerver/help/include/api-admin-only.md')
+            with open(file_name, 'r') as myfile:
+                file_content = myfile.read()
+            description_dict = description_dict.replace('{!api-admin-only.md!}', file_content)
```

Comment:
I would probably prefer for this to be a bit more generic; do you know how these are implemented in the normal markdown processor?  It's via `zerver.lib.bugdown.include.makeExtension`

Reply:
Done